blockchain/tx_pool_test.go

```diff
@@ -1970,9 +1983,9 @@ func TestDynamicFeeTransactionAccepted(t *testing.T) {
 	t.Parallel()
 	baseFee := big.NewInt(30)
 
-	pool, key := setupTxPoolWithConfig(eip1559Config)
```

Comment:
Don't we need to have a test case for eip1559 enabled and kip71 is disabled?

Reply:
It could be done with `setupTxPoolWithConfig(eip1559Config)` that means eip1559 enabled and kip71 disabled.

Reply:
You mean we have other test cases that having setupTxPoolWithConfig(eip1559Config)?

I'm asking because you replaced it with setupTxPoolWithConfig(kip71Config). It seems like you just removed the previous test cases and replaced it with a new test case with KIP-71.